liferay-echo/liferay-portal
# LPS-189847 Inform users about which experience or segment is being displayed

Motivation
=======
In a previous story we unified the way of simulating experiences and segments in the side panel.

However, when switching between options changes are not always obvious. This is why we need to better inform users about which experience/segment is being displayed and solve that it looks like changes are not being loaded. 
[Link to story or ticket](https://liferay.atlassian.net/browse/LPS-189847)

Proposed Solution
========
Display an alert in the Preview component when an event indicating that the selected segment or experience have been changed.

Steps to Verify:
----------------
1. Set feature.flag.LPS-186155=true in portal-ext.properties
1. Go to People -> Segments and create some segments
1. Edit the home page, create some experiences and publish
1. Open the simulation panel and check that there is an alert in the top of the Preview component indicating the segment or the experience selected.
1. Check that everything keep working as before without the feature flag


Screenshots (optional):
-----------------------
<img width="1781" alt="Captura de pantalla 2023-07-06 a las 11 32 47" src="https://github.com/liferay-echo/liferay-portal/assets/3276218/8b21d7d4-83a0-4764-9d25-3ff6214734d4">
